stack/conda.rst

```diff
@@ -24,6 +24,55 @@ Note that each of these other environments is maintained separately; see below.
 Requests for additions to ``rubin-env`` should be made via RFC.
 The DM-CCB must approve such requests.
 
+.. _rubin-env-switching-versions:
+
+Switching rubin-env versions
+----------------------------
+
+Occasionally, you may need to switch the version of ``rubin-env`` you are using.
+Below, you'll find instructions on how to view available versions and switch between them.
+You can view the current available env versions `on conda-forge <https://anaconda.org/conda-forge/rubin-env/files>`__.
+To see all available ``rubin-env`` in your terminal run the following:
+
+.. code-block:: bash
+
+   conda search -c nodefaults  -c conda-forge rubin-env
+
+This command will print out only production ready ``rubin-env`` versions.
+To change the ``rubin-env``:
+
+.. code-block:: bash
+
+   conda deactivate # make sure you do not run this inside an existing conda environment
+   ./bin/deploy -v 8.0.0 # or any other rubin env
+   source bin/envconfig -n lsst-scipipe-8.0.0
+   rebuild lsst_apps # This could be any branch or product you want to use
+
+
+.. _rubin-env-dev-switching-versions:
+
+Using developer rubin-env environments
+--------------------------------------
+
+You might also want to switch to rubin-env that are still under development.
+To do so, add the label conda-forge/label/rubin-env_dev as follows:
+
+.. code-block:: bash
+
+   conda search -c nodefaults  -c conda-forge rubin-env -c conda-forge/label/rubin-env_dev
+
+This command will print out all available ``rubin-env`` including ones that are still under development (rubin-env_dev).
+To change the ``rubin-env``:
+
+.. code-block:: bash
+
+   conda deactivate # make sure you do not run this inside an existing conda environment
```

Comment:
Maybe remind people here that it's better to start from a fresh shell whenever you run bin/deploy. I have found that very confusing things can happen if you don't do that (and we can remove the deactivate line if we do that).